docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -2,6 +2,20 @@
 Pulp Puppet 2.8 Release Notes
 =============================
 
+Pulp Puppet 2.8.4
+=================
+
+- If you were experiencing issues with modules installation you may want to run `pulp-manage-db`
```

Comment:
The running of pulp-manage-db is not optional they need to do it. I recommend removing this sentence altogether and having it start with the next sentence which focuses on informing the user about what is contained in the release and the optional steps they can take when they run `pulp-manage-db`.